ansible-scylla-monitoring/tasks/docker.yml

```diff
@@ -2,6 +2,12 @@
 - name: Populate service facts
   service_facts:
 
+- name: Add custom dockerd config
+  copy:
+    content: "{{ docker_daemon_custom_config }}"
+    dest: "/etc/docker/daemon.json"
+  become: true
```

Comment:
conditioning is missing: `when docker_daemon_custom_config is defined ...`

Reply:
Fixed